data/pigui/modules/hyperjump-planner.lua

```diff
@@ -340,7 +338,7 @@ function hyperJumpPlanner.onEnterSystem(ship)
 	updateHyperspaceTarget()
 end
 
-function hyperJumpPlanner.onGameEnd(ship)
+function hyperJumpPlanner.onGameEnd(_)
```

Comment:
Does onGameEnd actually take a `ship` parameter? AFAIK the related event fired by C++ is not passed any kind of context object, as game-end is a global event and not related to any one ship...

Reply:
Indeed!